Remove support for PNG generation

pkra opened the pull request:
Fixes #205.

pkra commented:
Should we make another release before v1.0 and other breaking changes (like #206)?

pkra commented:
This is marked WIP because of the open questions in #205.

pkra commented:
The only remaining question I had was whether https://github.com/mathjax/MathJax-node/blob/master/lib/mj-single.js#L587-L593 could/should be dropped or not. 

But that's a minor item I think (and could be done when #219 is decided) so I think this is ready to go for now.

pkra commented:
Ahem, this is not quite ready. I need to remove all other PNG-related things (readme, examples in bin etc)

pkra commented:
Ok, that's better. There are still some references in mj-page but that's for #206 to fix.

dpvc commented on README.md:
```diff
@@ -18,10 +18,7 @@ Use
-[Batik](http://xmlgraphics.apache.org/batik/download.html) library.  It 
-should be installed in the `batik` directory.  See the README file in that 
-directory for more details.
+These API's no longer produce PNG images but can easily be integrated into workflows using svg-to-png converters (e.g., png2svg, librsvg, batik).
```
Do you really mean `png2svg` here?  That sounds like the wrong direction.

pkra replied:
Thanks for catching that.

dpvc commented:
Just to be clear, this also removes not only the PNG generation, but the ability to produce image tags that use remote SVG files as well.  I assume you want to move that to a post-processing phase, but wanted to make sure that it was part of the plan, since it has not been discussed anywhere else and isn't in the PR title.

pkra commented:
> Just to be clear, this also removes not only the PNG generation, but the ability to produce image tags that use remote SVG files as well. I assume you want to move that to a post-processing phase, but wanted to make sure that it was part of the plan, since it has not been discussed anywhere else and isn't in the PR title.

Right. I actually remember not being able to figure out if this code was still relevant but then deciding that it also fits more naturally in post-processing. I can pull out the commit if you prefer (I have to rebase anyway).

dpvc commented:
>  I can pull out the commit if you prefer

No need.  Leave it in, I just wanted it to be in the record.

pkra commented:
@dpvc I rebased (and addressed your comment). I'm a little insecure as to the rebase (lots of changes within mj-single) but I went through it twice and I didn't spot any bad changes from the resolution.

dpvc commented:
There rebase looks good.  I'll merge.
